_includes/footer.html

```diff
@@ -1,2 +1,8 @@
 {% assign footer = site.data.footer %}
-{% include components/footer.html %}
+
+
+{% if footer.type == 'medium' %}
+  {% include components/footer--medium.html %}
+{% else %}
+{% include components/footer--medium.html %}
```

Comment:
Is this in preparation for other options to be added?

Reply:
Yeah, I was testing a few things. I'm going to remove it for the time being and do that work in a separate PR.